contracts/CombinedChainlinkOracle.vy

```diff
@@ -0,0 +1,33 @@
+struct LatestRoundData:
+    round_id: uint80
+    answer: int256
+    started: uint256
+    updated: uint256
+    answered_round: uint80
+
+interface ChainlinkOracle:
+    def latestRoundData() -> LatestRoundData: view
+    def decimals() -> uint256: view
+
+SCALE: constant(int256) = 10**18
+
+yfi_oracle: public(immutable(ChainlinkOracle)) # YFI/USD
+eth_oracle: public(immutable(ChainlinkOracle)) # ETH/USD
+decimals: public(constant(uint256)) = 18
+
+@external
+def __init__(_yfi_oracle: address, _eth_oracle: address):
+    yfi_oracle = ChainlinkOracle(_yfi_oracle)
```

Comment:
probably safer to hardcode them here since we have them already to avoid operational mistake in deployment?